ffi-cdecl/mupdf_decl.c

```diff
@@ -102,33 +108,20 @@ cdecl_func(mupdf_load_outline)
 cdecl_func(fz_drop_outline) // NOTE: libk2pdfopt uses old fz_free_outline symbol
 
 /* structured text */
-cdecl_type(fz_text_style)
-cdecl_struct(fz_text_style_s)
-
-cdecl_type(fz_text_char)
-cdecl_struct(fz_text_char_s)
-
-cdecl_type(fz_text_span)
-cdecl_struct(fz_text_span_s)
-
-cdecl_type(fz_text_line)
-cdecl_struct(fz_text_line_s)
-
-cdecl_type(fz_text_sheet)
-cdecl_struct(fz_text_sheet_s)
-cdecl_func(mupdf_new_text_sheet)
-cdecl_func(fz_drop_text_sheet) // NOTE: libk2pdfopt uses old fz_free_text_sheet symbol
```

Comment:
The fz_drop_text_sheet function no longer exists.

I don't think those comments are relevant anymore because libk2pdfopt is built without using MuPDF (HAVE_MUPDF_LIB).